src/matcha_ml/state/remote_state_manager.py

```diff
@@ -224,6 +225,10 @@ def _write_matcha_config(
             )
         )
 
+    def _remove_matcha_config(self) -> None:
+        """Remove the matcha.config.json file after destroy full is run."""
+        os.remove(self.config_path)
```

Comment:
Is the case where this file doesn't exist handled? At the moment, this will throw an `FileNotFoundError`